.gitignore

```diff
@@ -629,3 +629,7 @@ FodyWeavers.xsd
 *.tlog
 
 # End of https://www.toptal.com/developers/gitignore/api/r,python,macos,windows,visualstudio,visualstudiocode
+
+javascript-package/node_modules/
+javascript-package/dist/
+javascript-package/coverage/
```

Comment:
might want to also put the package-lock.json file in this, thoughts?

Reply:
I think we should probably leave it in, `package-lock.json` is [intended to be version controlled](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json).